starter-code/memory.js

```diff
@@ -60,4 +89,56 @@ $(document).ready(function(){
 
   // Add all the divs to the HTML
   document.getElementById('memory_board').innerHTML = html;
+
+  $(".card .front").hide();
+
+  // my actual code starts here, M
+
+  function updateScore(){
+    $("#pairs_clicked").text(memoryGame.pairsClicked);
+    $("#pairs_guessed").text(memoryGame.correctPairs);
+  }
+  var lastCard;
+  $('.back').on('click', function(){
+    console.log(memoryGame.selectedCards.length);
+    if (memoryGame.selectedCards.length == 2) {
+      console.log("STOP!!!!!!!")
+      return;
+    }
+    var currentImgValue = $(this).attr("id");
+
+    var cardsMustRestart = memoryGame.selectCard(currentImgValue);
+
+    if (memoryGame.selectedCards.length !== 0){
+      lastCard = $(this);
+    }
+    else{
+      updateScore();
+    }
+
+    var backside = $(this);
+    var frontside = $(this).siblings();
```

Comment:
remember that if you write `$(this)` more than once, you can write `var me = $(this)` and reuse it